include/deal.II/lac/full_matrix.h

```diff
@@ -975,6 +975,33 @@ class FullMatrix : public Table<2, number>
                  const bool                transpose_D = false,
                  const number              scaling     = number(1.));
 
+
+  /**
+   * @brief Compute the Kronecker product of two matrices.
+   *
+   * This function computes the Kronecker product of two matrices A and B, and
+   * stores the result in the current matrix. The Kronecker product of an
+   * m x n matrix A and a p x q matrix B is an (m*p) x (n*q) matrix defined as:
+   *
+   * ```
+   * A ⊗ B = | a11*B   a12*B   ...   a1n*B   |
+   *           | a21*B   a22*B   ...   a2n*B   |
+   *           |  ...     ...     ...     ...   |
+   *           | am1*B   am2*B   ...   amn*B   |
```

Comment:
Can you align the rows. Make sure that you are using blank spaces and  not tabs (the length of a tab depends on the text editor).

Reply:
Formatting fixed